src/tmx/TmxCtl/src/lib/PluginStatus.cpp

```diff
@@ -558,6 +562,58 @@ bool TmxControl::user_info(bool showPassword)
 	return true;
 }
 
+bool TmxControl::hashed_info(pluginlist &, ...)
+{
+	if (!checkPerm())
+		return false;
+	return hashed_info();
+}
+
+bool TmxControl::hashed_info()
+{
+	string query = HASHED_USER_QUERY;
+	if (_opts->count("password") == 0 || (*_opts)["password"].as<string>() == "")
+		return NULL;
+	query += " WHERE IVP.user.password = SHA2(?, 256)";
+
+	try
+	{
+		PLOG(logDEBUG1) << "Executing query " << query;
+
+		_output.get_storage().get_tree().clear();
+
+		std::string pwd = _pool.GetPwd();
+		DbConnection conn = _pool.Connection("tcp://127.0.0.1:3306","IVP", pwd, "IVP");
```

Comment:
No need to pass the password in here.  You can just use _pool.Connection() to get a valid connection.

Reply:
Leaving as is, but will be reworked later.